src/table/TableBodyWrapper.jsx

```diff
@@ -12,12 +12,22 @@ const useStyles = makeStyles({
     fontSize: props.fontSize,
     color: props.fontColor,
   }),
+  hoverTableRow: (props) => ({
+    '&&:hover': {
+      backgroundColor: props.hoverBackGroundColor,
+      '& td': {
+        color: props.hoverFontColor,
+      },
+    },
+  }),
 });
 
-export default function TableBodyWrapper({ tableData, constraints, selectionsAPI, layout, theme }) {
+export default function TableBodyWrapper({ tableData, constraints, selectionsAPI, layout, theme, cellLockedImage }) {
   const { rows, columns } = tableData;
-  const classes = useStyles(getBodyStyle(layout, theme));
+  const hoverEffect = layout.components?.[0]?.hoverEffect;
```

Comment:
It's actually:

```suggestion
  const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
```

Reply:
<img width="185" alt="Screenshot 2021-02-22 at 10 22 45" src="https://user-images.githubusercontent.com/4471489/108688552-56afb800-74f8-11eb-9fcb-3c667d8de23e.png">
From what I see in sense-client, the hoverEffect is not within the content.

Reply:
it is, that } is closing the fontColor property.

Reply:
yes, you are right!